python/pyspark/testing/utils.py

```diff
@@ -28,6 +28,7 @@
 
 have_scipy = False
 have_numpy = False
+have_memory_profiler = False
```

Comment:
Can't we reuse `pyspark.profiler.has_memory_profiler`?

Reply:
Sounds good.